fenix/app/src/main/java/org/mozilla/fenix/browser/TabPreview.kt

```diff
@@ -50,6 +53,12 @@ class TabPreview @JvmOverloads constructor(
 
     override fun onLayout(changed: Boolean, left: Int, top: Int, right: Int, bottom: Int) {
         super.onLayout(changed, left, top, right, bottom)
+
+        store.state.selectedTab ?.let {
```

Comment:
nit: looks like there's a missing space here
```suggestion
        store.state.selectedTab?.let {
```